daft/runners/ray_runner.py

```diff
@@ -877,7 +877,7 @@ def _run_plan(
 
         inflight_tasks: dict[str, PartitionTask[ray.ObjectRef]] = dict()
         inflight_ref_to_task: dict[ray.ObjectRef, str] = dict()
-        pbar = ProgressBar(use_ray_tqdm=self.use_ray_tqdm)
```

Comment:
BTW do we need to worry that any of these changes will break the current ray runner? Since we might have a release or 2 where the ray runner is still the default